resources/views/users/edit-add.blade.php

```diff
@@ -112,13 +112,3 @@
         </div>
     </div>
 @stop
-
```

Comment:
Please explain why this was removed.  It doesn't seem related to the rest of the changes.